specification/dimensions/commitmentname.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Name
+
+A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
```

Comment:
Hmm tying in with my comment from CommitmentId - should we be duplicating this on every Cxxx column? Glossary?

Reply:
For now, yes.